### ISSUE TITLE
Run configurations

### ISSUE THREAD

**Issue by boniatillo-com** [boniatillo-com/PhaserEditor#48]
(from @samsstuff)

...alright if you try and use the run menu there is nothing to choose to run it with (I never use that anyhow) but the globe still works fine which is what I always use anyhow, just in case anyone ask about that, it pops up external tools with nothing but you can config it to load anything you want in case anyone wants to pick a different browser or anything else.

**Comment by boniatillo-com** [comment]
Yes, actually I was thinking about it, to create run configurations to run the projects in an external browser or in an internal one.

It came to my mind after this issue:

https://github.com/boniatillo-com/PhaserEditor/issues/45

**Comment by samsstuff** [comment]
sounds good
